bigquery/google/cloud/bigquery/magics.py

```diff
@@ -300,7 +300,7 @@ def _run_query(client, query, job_config=None):
     while True:
         print("\rQuery executing: {:0.2f}s".format(time.time() - start_time), end="")
         try:
-            query_job.result(timeout=0.5)
+            query_job.result(timeout=0.5, max_results=max_results)
```

Comment:
Since we aren't actually returning the results in this line (just waiting for the query to finish), we don't need to pass `max_results` here. I believe that means we can remove the `max_results` parameter from `_run_query` as well.

Reply:
Removed in f88ffe8